frontend/lib/src/components/shared/BaseColorPicker/BaseColorPicker.test.tsx

```diff
@@ -35,147 +34,110 @@ const getProps = (
-      opacity: "",
-    })
-
-    expect(chromePickerWrapper.prop("color")).toEqual("#000000")
-  })
+    const props = getProps()
+    render(<BaseColorPicker {...props} />)
 
-  it("supports hex shorthand", () => {
-    wrapper.find(UIPopover).simulate("click")
+    const colorBlock = screen.getByTestId("stColorBlock")
+    fireEvent.click(colorBlock)
 
-    // @ts-expect-error do not need change event added
-    colorPickerWrapper.prop("onChange")({
-      hex: "#333",
-    })
+    expect(colorBlock).toHaveStyle("background-color: #000000")
 
-    expect(
-      wrapper
-        .find(UIPopover)
-        .renderProp("content")({ close: () => {} })
-        .find(ChromePicker)
-        .prop("color")
-    ).toEqual("#333")
+    const colorInput = screen.getByRole("textbox")
+    expect(colorInput).toHaveValue("#000000")
   })
 
-  it("should update the widget value when it's changed", () => {
-    const newColor = "#E91E63"
-    wrapper.find(UIPopover).simulate("click")
-
-    // @ts-expect-error do not need change event added
-    colorPickerWrapper.prop("onChange")({
-      hex: newColor,
-    })
+  it("supports hex shorthand", () => {
+    const props = getProps()
+    render(<BaseColorPicker {...props} />)
 
-    expect(
-      wrapper
-        .find(UIPopover)
-        .renderProp("content")({ close: () => {} })
-        .find(ChromePicker)
-        .prop("color")
-    ).toEqual(newColor)
-  })
+    const colorBlock = screen.getByTestId("stColorBlock")
+    fireEvent.click(colorBlock)
 
-  it("should disable alpha property for now", () => {
-    wrapper.find(UIPopover).simulate("click")
-    expect(colorPickerWrapper.prop("disableAlpha")).toStrictEqual(true)
-  })
-})
+    const colorInput = screen.getByRole("textbox")
+    fireEvent.change(colorInput, { target: { value: "#333" } })
 
-describe("ColorPicker widget with optional params", () => {
-  it("renders with showValue", () => {
-    const props = getProps({ showValue: true })
-    const wrapper = shallow(<BaseColorPicker {...props} />)
-    expect(wrapper.find("StyledColorValue").exists()).toBe(true)
+    expect(colorInput).toHaveValue("#333333")
+    expect(colorBlock).toHaveStyle("background-color: #333333")
   })
 
-  it("renders without showValue", () => {
+  it("should update the widget value when it's changed", () => {
     const props = getProps()
-    const wrapper = shallow(<BaseColorPicker {...props} />)
-    expect(wrapper.find("StyledColorValue").exists()).toBe(false)
-  })
+    render(<BaseColorPicker {...props} />)
 
-  it("should render TooltipIcon if help text provided", () => {
-    const props = getProps({ help: "help text" })
-    const wrapper = shallow(<BaseColorPicker {...props} />)
-    expect(wrapper.find("TooltipIcon").prop("content")).toBe("help text")
-  })
-})
+    const newColor = "#E91E63"
+    const colorBlock = screen.getByTestId("stColorBlock")
+    fireEvent.click(colorBlock)
 
-describe("ColorPicker error handler", () => {
```

Comment:
These two tests essentially check the behavior of the `componentDidCatch` method and aren't in line with RTL philosophy (haven't been able to find anything like `simulateError`) so removing unless there are any objections